### DISCUSSION TITLE
Azure Log-based Metrics support

### DISCUSSION THREAD

**Discussion by mpostument** [grafana/grafana#62288] [Data Sources]
Azure support Log-based Metrics which is currently not supported in grafana. By any chance this feature may be implemented in nearest future?

More details in this issue https://github.com/grafana/grafana/issues/60450#issuecomment-1403897521

**Comment by jrgcubano** [comment]
@aangelisc Any news on this? Regards

**Reply by aangelisc** [reply]
Hi @jrgcubano, there's no update on this as of yet. If this gets added to our roadmap I'll be sure to update the discussion 😊

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is being addressed here https://github.com/grafana/grafana/issues/82913 Feel free to make an [issue](https://github.com/grafana/grafana/issues) if I'm mistaken to continue the discussion.
